templates/cassiopeia/html/layouts/chromes/noTitle.php

```diff
@@ -0,0 +1,26 @@
+<?php
+/**
+ * @package     Joomla.Site
+ * @subpackage  Templates.crocosmia
+ *
+ * @copyright   Copyright (C) 2005 - 2019 Open Source Matters, Inc. All rights reserved.
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+defined('_JEXEC') or die;
+
+$module  = $displayData['module'];
+$params  = $displayData['params'];
+$attribs = $displayData['attribs'];
+
+$modulePos   = $module->position;
+$moduleTag   = $params->get('module_tag', 'div');
+$modId 		 = 'mod-' . $module->id;
```

Comment:
```suggestion
$modulePos = $module->position;
$moduleTag = $params->get('module_tag', 'div');
$modId     = 'mod-' . $module->id;
```
See my previous review comments.

Reply:
See above.

(and I don't know what 'Resolve conversation' underneath these posts means)

Reply:
resolve conversation you should use when you have applied a requested change by using the button for applying that change, or when you have applied changes in another way which resolve a conversation. After a conversation has been resolved, it is hidden by default, and requested changes are not counted anymore.